test/unit/teacher_set_test.rb

```diff
@@ -163,6 +163,23 @@ class TeacherSetTest < ActiveSupport::TestCase
 
   end
 
+  describe 'test update_subjects method' do
+    # Case: 1
+    it 'test with empty subjects' do
+      subject_name_array = []
+      resp = @teacher_set.update_subjects_via_api(subject_name_array)
+      assert_nil(resp)
+    end
+
+    # Case: 2
+    it 'update subjects' do
+      subject_ids = subjects(:subjects_one)
+      subject_name_array = ["Authors, Russiantest", "Social"]
+      resp = @teacher_set.update_subjects_via_api(subject_name_array)
+      assert_equal(subject_ids.id, resp[0])
```

Comment:
Presumably

```      assert_equal(subjects(:subjects_two).id, resp[1])```

would also pass, right?
I'd add that as well, for completeness (and to stop me from wondering if it's true).